From df1f4bb2b0281b192156b0ae327cebb0758305c7 Mon Sep 17 00:00:00 2001
From: Nicholas Little <nicholas.john.little@gmail.com>
Date: Tue, 28 May 2024 01:56:53 +0100
Subject: [PATCH] hwdb: Lenovo IdeaPad Z500 Touchpad Toggle (#33039)

Map scancode f3 to KEY_F21.

(cherry picked from commit f4434c61b91a152f71f760f8ad754ba6b92cade5)

Related: RHEL-5950
---
 hwdb.d/60-keyboard.hwdb | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 5af2303e98..3eb323f0e8 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -1060,6 +1060,8 @@ evdev:atkbd:dmi:*:svnLENOVO:*:pvrLenovoYoga300-11IBR:*
 
 # Lenovo Y50-70
 evdev:atkbd:dmi:bvn*:bvr*:svnLENOVO*:pn*20378*:pvr*
+# Lenovo IdeaPad Z500
+evdev:atkbd:dmi:bvn*:bvr*:svnLENOVO*:pn*5931*:pvr*
  KEYBOARD_KEY_f3=f21                                    # Fn+F6 (toggle touchpad)
 
 # V480
